ocs_ci/helpers/dr_helpers.py

```
@@ -4553,6 +4553,13 @@ def validate_application_odf_cli(
             )
             return None
 
+    if config.MULTICLUSTER.get("multicluster_mode") == constants.MDR_MODE:
+        logger.info(
+            f"Skipping ODF CLI DR {action} for DRPC '{drpc_name}': "
+            f"MDR environment does not require application validation"
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)